test/parallel/test-http-outgoing-properties.js

```diff
@@ -51,3 +51,28 @@ const OutgoingMessage = http.OutgoingMessage;
   msg.write('asd');
   assert.strictEqual(msg.writableLength, 7);
 }
+
+{
+  const server = http.createServer(function(req, res) {
+    res.end();
+    res.on('finish', () => {
+      server.close();
+    });
+  });
+
+  server.listen(0);
+
+  server.on('listening', function() {
```

Comment:
The callback here should be wrapped in a `common.mustCall()`.

Reply:
@mscdex  Thank you for your patience guidance,  I submitted the code just now. Please check it again for me.